docs/starters.yml

```diff
@@ -1,3 +1,22 @@
+- url: https://gatsby-starter-carraway.netlify.com/
+  repo: https://github.com/endymion1818/gatsby-starter-carraway
+  description: a Gatsby starter theme with Accessibility features, TypeScript, Jest, some basic UI elements, and a CircleCI pipeline
+  tags:
+    - TypeScript
+    - Pagination
+    - Search
+    - Tests
```

Comment:
`Tests` is not an available category. `Testing` is. Please consider adding that.

Reply:
👍  Thanks, I've amended my PR